flash_update/test_views.py

```diff
@@ -5,13 +5,15 @@
 
 from main.test_case import APITestCase
 import api.models as models
-from flash_update.models import FlashUpdate, FlashEmailSubscriptions, FlashGraphicMap
+from flash_update.models import FlashUpdate, FlashEmailSubscriptions, FlashGraphicMap, FlashUpdateShare
 from flash_update.factories.flash_update import (
     FlashUpdateFactory,
     FlashGraphicMapFactory,
-    FlashActionFactory
+    FlashActionFactory,
+    DonorFactory,
+    DonorGroupFactory
```

Comment:
Let's always add `,` for the last item in the list as well. This creates a better diff for future changes.